index.html

```diff
@@ -146,8 +159,10 @@ <h2 class="card-name">Multi-post stories</h2>
       </ul>
       <button class="sec-button">see project</button>
     </div>
+    </div>
   </section>
   <section class="about">
+    <div class="new-about">
     <h2 class="title-about">About<br>Myself</h2>
     <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and
       experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] [ Optional ] As the requirement you should stick to the design as much as possible that is font, color, and spacing. You don't have spacing before `About section`, If you see the design you will find the spacing so please implement it and also see the image below for reference.
### Your design
![yourrrrrrrrrr](https://user-images.githubusercontent.com/81727988/194520718-94ffbe7b-2520-4175-a85b-e2058fb5e916.png)

 
### Figma design
![figmaaaaaaa](https://user-images.githubusercontent.com/81727988/194520572-32cca7c3-d186-4cd7-bbac-d42ed14dd01e.png)